modules/grafana/local/provisioning/datasources/automatic.yml

```diff
@@ -0,0 +1,418 @@
+apiVersion: 1
```

Comment:
If this is here, is it needed in the other repository too?

Reply:
Removed from the other repository